samples/javascript_nodejs/21.corebot-app-insights/index.js

```diff
@@ -65,25 +79,21 @@ const userState = new UserState(memoryStorage);
 const { LuisAppId, LuisAPIKey, LuisAPIHostName } = process.env;
 const luisConfig = { applicationId: LuisAppId, endpointKey: LuisAPIKey, endpoint: `https://${ LuisAPIHostName }` };
 
-const luisRecognizer = new FlightBookingRecognizer(luisConfig);
+const luisRecognizer = new FlightBookingRecognizer(luisConfig, telemetryClient); 
 
 // Create the main dialog.
 const bookingDialog = new BookingDialog();
```

Comment:
Need to add the const  `BOOKING_DIALOG`

```Javascript
const bookingDialog = new BookingDialog(BOOKING_DIALOG);
```

This is how it is done in Sample 13.CoreBot and should not be changed when updating to enable telemetry.

Note also the const declaration is currently still in `index.js` on line 22:

```JavaScript
const BOOKING_DIALOG = 'bookingDialog';
```

but with the current implementation is not being used.

Reply:
Amended. Thanks.